### DISCUSSION TITLE
Где размещать информацию о том, почему были приняты такие решения

### DISCUSSION THREAD

**Discussion by eugenekogan-spb** [fhir-ru/core#255] [Минимальный набор артефактов]
Решения по профилированию принимаются в рамках устных обсуждений на встречах или письменных в дискуссиях. Иногда это запутанные, пересекающиеся обсуждения, и давать просто ссылку на несколько дискуссий и встреч - негуманно. Возможно, следует по важным задачам делать краткое описание, какие варианты рассмотрены и почему в итоге принято такое решение.

**Вариант 1. Оставить в дискуссиях, формировать текст в карточке дискуссии** 
Плюсы
удобно - если для понимания не хватило выжимки в карточке дискуссии, промотай ниже и посмотри подробности
Минусы
Получается три места просмотра информации - руководство, страницы сайта сообщества, дискуссии. Не очевидно, что где искать.

**Вариант 2. На странице сайта сообщества с данным артефактом**

**Вариант 3. Специальными страницами в скопмилированном руководстве**

**Comment by Vrodvi** [comment]
При работе полностью на стандартных инструментах я бы предложил использовать "Вариант 3". Инструментарий SUSHI позволяет формировать подобные страницы и органично включать их в состав руководства. К примеру, страницы с пояснениями по принятым решениям по профилированию имеются в зарубежных руководствах - US Core, mCode, и других:
<img width="144" alt="2023-12-08_15h37_33" src="https://github.com/fhir-ru/core/assets/92928137/3615f562-bacb-4941-bd03-8a5685203ab4">
<img width="144" alt="image" src="https://github.com/fhir-ru/core/assets/92928137/8990fa60-dee1-4499-bcad-084bf5b7b938">

Но поскольку, в качестве основного инструмента именно документирования выбран Zendoc, то, всё же, предпочтительным место будет именно он, т.е. рекомендуется использовать **"Вариант 2."**.

При этом страницы с описаниями на сайте сообщества всё ещё можно снабжать ссылками на одну или более дискуссий, в рамках которых, проходило обсуждение и принимались решения, но подобного рода ссылки желательно не делать очень бросающимися в глаза.

**Reply by Vrodvi** [reply]
Также обсуждалось, что в стандартном руководстве по реализации, можно в описательной части профиля (в заголовке или подвале) давать ссылку на страницу Zendoc, где собраны все решения.
Таким образом, получится три уровня описания - формальное описание в виде стандартного руководства, затем из него ссылка на Zendoc с развёрнутой пояснительной информацией, и из неё, при необходимости, ссылки на дискуссии.

**Reply by eugenekogan-spb** [reply]
Поддерживаю. Ждем реализации заголовков/подвалов.
